cmd/img_mgmt/src/img_mgmt.c

```diff
@@ -241,7 +241,7 @@ img_mgmt_find_by_ver(struct image_version *find, uint8_t *hash)
     int i;
     struct image_version ver;
 
-    for (i = 0; i < 2; i++) {
+    for (i = 0; i < 2*CONFIG_UPDATEABLE_IMAGE_NUMBER; i++) {
```

Comment:
The config will break builds for other systems, as it is used in common code.
I think that intermediate preprocessor name should be provided that will be set to 1 when `CONFIG_UPDATE_IMAGE_NUMBER` is not set.

Reply:
Yeah already commented that I think just adding a
```C
#ifndef CONFIG_UPDATEABLE_IMAGE_NUMBER
#define CONFIG_UPDATEABLE_IMAGE_NUMBER 1
#endif
```
should be sufficient.